dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -1503,8 +1510,7 @@ public Identifier getRelatedIdentifier(Contentlet contentlet,String relationship
     }
 
     private List<Contentlet> getRelatedChildren(final Contentlet contentlet, final Relationship rel,
-            final User user, final boolean respectFrontendRoles, int limit, int offset,
```

Comment:
Should this use the db to getRelatedChildren?

Reply:
What do you mean @wezell ? It considers the value set for the property `GET_RELATED_CONTENT_FROM_DB`